changelog.md

```diff
@@ -159,6 +159,8 @@ proc mydiv(a, b): int {.raises: [].} =
   The reason for this is that `DivByZeroDefect` inherits from `Defect` and
   with `--panics:on` `Defects` become unrecoverable errors.
 
+- Added `thiscall` calling convention as specified by Microsoft, mostly for hooking purpose
```

Comment:
What's a "hooking purpose"?

Reply:
kind of detours/api hooking/patching. cuz I dont see a use case of `thiscall` in real life for Nim

Reply:
It's ok.